keras/callbacks.py

```diff
@@ -573,7 +575,11 @@ def on_epoch_begin(self, epoch, logs=None):
         if not isinstance(lr, (float, np.float32, np.float64)):
             raise ValueError('The output of the "schedule" function '
                              'should be float.')
-        K.set_value(self.model.optimizer.lr, lr)
+        if lr < float(K.get_value(self.model.optimizer.lr)):
```

Comment:
This conditional should be removed, this is not the expected behavior

Reply:
What would be your recommended way to solve broken cifar10_resnet.py? Currently,  `LearningRateScheduler` always overrides `ReduceLROnPlateau`, and there is no obvious way to solve it, other that writing your own `LearningRateScheduler`. Maybe `LearningRateScheduler` should have a parameter `force` defaulting to `True`? I can also print a warning if LR is increased, either when `verbose` is in effect or unconditionally.

Reply:
Removed. Found a much better way.